bgpd/bgp_route.c

```diff
@@ -3157,6 +3157,13 @@ int bgp_update(struct peer *peer, struct prefix *p, uint32_t addpath_id,
 		goto filtered;
 	}
 
+	if (pi && pi->attr &&
```

Comment:
What happens if the user has: `set table 99` and then enters in quick succession `set table 44` `set table 300` `set table 9900`?

Do we handle this situation correctly?

Reply:
the table 9900 will be used to export bgp entries.
this is because the new value is written in bgp route-map, but will be applied a little bit alter to incoming bgp route entries. ( this can be checked with neighbor soft discovery, but also if you do clear bgp ).
so this is handled correctly.